content/en/contribute.md

```diff
@@ -3,74 +3,121 @@ title: Contribute to NumPy
+development, these
+[issues](https://github.com/numpy/numpy.org/issues?q=is%3Aissue+is%3Aopen+label%3Adesign)
+list some of our unmet needs -- and feel free to share your own ideas.
 
 
 ### Graphic design
 
-We would love to get contributions from graphic designers! This website could be further improved with new graphics. And we'd be very interested in making, e.g., our tutorials more visually engaging.
+We've barely begun to think of all the contributions a graphic designer can
+make on this site. Our docs are parched for illustration; our growing website
+craves images -- opportunities abound.
 
 
 ### Translating website content
 
-We aim to translate this [numpy.org](https://numpy.org) website into multiple languages, to make NumPy more accessible to users who are more comfortable in their own language than in English. We are still putting the infrastructure in place to make this happen. We already welcome volunteers though! Please see [the high-level plan](https://numpy.org/neps/nep-0028-website-redesign.html#translation-multilingual-i18n) and comment on [this GitHub issue](https://github.com/numpy/numpy.org/issues/55) if you are interested to help out.
-
-
-### Fundraising
-
-NumPy has been an all-volunteer project for most of its history, however, with the continuous growth of our user base we feel the need for funding to keep the project healthy. [This SciPy'19 talk](https://www.youtube.com/watch?v=dBTJD_FDVjU) attempts to quantify the impact of recently received funding and the needs of the project. We have a number of ideas about how to obtain funding (and of course welcome more good ideas!), these all take time to execute though. Fundraising is also a skill that not many current team members have - we'd love your help!
+We plan multiple translations of [numpy.org](https://numpy.org) to make NumPy
+accessible to users in their native language. Volunteer translators, are, of
+course, at the heart of this effort.  See
+[here](https://numpy.org/neps/nep-0028-website-redesign.html#translation-multilingual-i18n)
+for background; comment on [this GitHub
+issue](https://github.com/numpy/numpy.org/issues/55) to sign up.
 
 
 ### Community coordination and outreach
 
-Between community calls, our Twitter account, improving our documentation on processes and community topics, organizing sprints, a newsletter, and perhaps in the future a blog, there is a lot of organizing and outreach to do. We'd love to get more people involved in this important aspect of the project!
+Through community contact we share our work more widely and learn where we're
+falling short. We'd love to get more people involved in efforts like our
+[Twitter](https://twitter.com/numpy_team) account, organizing NumPy [code
+sprints](https://scisprints.github.io/), a newsletter, and perhaps a blog.
+
+### Fundraising
 
+For most of its history NumPy was all-volunteer, but as the user base grew,
+paid staffers were added to ensure the project stayed healthy. [This SciPy'19
```

Comment:
I'm not sure about the "paid staffers" and past tense here. It makes it sound like a solved problem, which it very much is not. The main source of funding was the BIDS grant, and that's running out in October. And even with that, we were still short on maintenance power to actually reduce the backlog of PRs for example. Could you try to rephrase?

Reply:
Thanks for clarifying this. Revised version makes clear that funding needs are ongoing.